pages/mi/mi-10-steps.md

```diff
@@ -0,0 +1,116 @@
+## Step 0 - Prerequisites
+
+The following hardware is required for the internship:
+
+1. Laptop or computer with minimum 8GB of RAM
+2. Android device (phone or tablet) with minimum 2GB RAM and at least Android 6 "Marshmallow"
+3. Internet connection
+
+## Step 1 - Git markdown
+
+- Follow the instructions on the GitHub and Markdown page. Make sure that you've linked to your github.io and pull request link on the Gitter chat (https://YourUserName.github.io and LinkToYourPullRequest).
```

Comment:
Remove mention of gitter, use discord

Reply:
Which discord channel will be used by interns for communication